recipes/libaesgm/all/conanfile.py

```diff
@@ -0,0 +1,72 @@
+from conan import ConanFile
+from conan.tools.files import get, copy, load, save
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain, cmake_layout
+import os
+
+
+required_conan_version = ">=1.53.0"
+
+class LibaesgmConan(ConanFile):
+    name = "libaesgm"
+    description = "Library implementation of AES (Rijndael) cryptographic methods"
+    license = "BSD"
```

Comment:
```suggestion
    license = "LicenseRef-libaesgm-BSD"
```

It looks like BSD-2, but is more simplified, so it's not an official SPDX license. Let's follow https://github.com/conan-io/conan-center-index/blob/master/docs/adding_packages/conanfile_attributes.md#license-attribute